javaagent-exporters/jaeger-thrift/jaeger-thrift.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     exclude group: 'io.opentelemetry', module: 'opentelemetry-sdk'
     exclude group: 'io.opentelemetry', module: 'opentelemetry-api'
   }
+  implementation group: 'io.grpc', name: 'grpc-netty-shaded', version: '1.30.2'
```

Comment:
is this needed?

Reply:
LOL I added it because I thought you said it was.  🤣  FWIW the smoke tests passed without it....should I take it back out?

Reply:
oh, sorry, I don't think thrift uses grpc, so would probably remove it to avoid confusion (otlp and jaeger do, which is why those modules have it)

Reply:
Got it.  I'll pull this back out.  Thanks!